src/hpack/table.py

```diff
@@ -218,14 +220,14 @@ def _shrink(self):
         self._current_size = cursize
 
 
-def _build_static_table_mapping():
+def _build_static_table_mapping() -> dict[bytes, tuple[int, dict[Any, Any]]]:
```

Comment:
Surprised to see `dict[Any, Any]` here. Can they really be anything or was this an escape hatch to reduce further complexity in types?

Reply:
You correctly identified the escape hatch! I wasn't yet able to derive the correct type from looking at the tests and code. Happy to change it to something more specific!

Reply:
Finally found the correct type - will be fixed in the next push!